src/PowerSystems2PRAS.jl

```diff
@@ -396,33 +302,29 @@ function generate_pras_system(
     λ_gen = Matrix{Float64}(undef, n_gen, s2p_meta.N)
     μ_gen = Matrix{Float64}(undef, n_gen, s2p_meta.N)
 
+    #FIXME
     for (idx, g) in enumerate(gen)
-        if (
-            lump_region_renewable_gens && (
-                PSY.get_prime_mover_type(g) == PSY.PrimeMovers.WT ||
-                PSY.get_prime_mover_type(g) == PSY.PrimeMovers.PVe
-            )
-        )
+        if haskey(PSY.get_ext(g), "region_gens")
             reg_gens_DA = PSY.get_ext(g)["region_gens"]
             gen_cap_array[idx, :] =
                 round.(
```

Comment:
This loop gives me a headache to follow what's up. We should abstract this into a function because it impossible to check if the logic is reasonable. @daniel-thom you might have some insight on the use of these functions for time series here.

Reply:
This is a mix of loop for the weird indexing PRAS needs and the lumped renewable generation. Later, I think we should spit the indexing into its own logic somehow later.